Marlin/src/lcd/menu/menu_filament.cpp

```diff
@@ -349,8 +351,13 @@ void menu_advanced_pause_option() {
   #if LCD_HEIGHT > 2
     STATIC_ITEM(MSG_FILAMENT_CHANGE_OPTION_HEADER, true, false);
   #endif
-  MENU_ITEM(function, MSG_FILAMENT_CHANGE_OPTION_RESUME, lcd_advanced_pause_resume_print);
   MENU_ITEM(function, MSG_FILAMENT_CHANGE_OPTION_PURGE, lcd_advanced_pause_extrude_more);
+  #if ENABLED(FILAMENT_RUNOUT_SENSOR)
+    if (runout.filament_ran_out)
+      MENU_ITEM_EDIT_CALLBACK(bool, MSG_RUNOUT_SENSOR_ENABLE, &runout.enabled, runout.reset);
+    else
+  #endif
```

Comment:
@InsanityAutomation — I don't understand the purpose of this menu item. After filament runs out, is the sensor being disabled? Is the runout sensor interfering with filament change?

Reply:
The intention is since we are disallowing resume when the sensor is still triggered, allow the user to disable it in the case of a faulty sensor. This way they are not stuck losing the print. In the past they could quickly get to the configuration menu before it tripped again depending on settings, but this is much more intuitive.

Many optical sensors have issues only with specific filament, encoder types may have issues with soft flexible materials, if we are going to block continue (which I do agree is a good move) until the sensor state is good again, I feel this is a necessary recovery method.

Reply:
I see. Since this is a failsafe, let's move the option to the bottom.

Reply:
@thinkyhead are you OK with the above? If so ill mark this as resolved. Id like to build off this menu stackup while configuring the prompts and just want to be sure this is going to be acceptable.